src/oge/helpers.py

```diff
@@ -349,53 +349,56 @@ def assign_fleet_to_subplant_data(
 
     # check that there is no missing ba or fuel codes for subplants with nonzero gen
     # for CEMS data, check only units that report positive gross generaiton
-    if "gross_generation_mwh" in subplant_data.columns:
-        missing_fleet_keys = subplant_data[
-            (
+    if (
```

Comment:
Needed to add logic to not do these checks if neither gross generation nor net generation is in the df